abi/abi/HEVM.sol

```diff
@@ -180,6 +180,7 @@ writeJson(string, string)
 writeJson(string, string, string)
 parseJson(string)(bytes)
 parseJson(string, string)(bytes)
+parseJsonKeys(string, string)(string[])
```

Comment:
should we add a `parseJsonKeys(string)(string[])` to parse the root keys? might be better UX than passing `$` as the key, but I don't do tons of jsonpath stuff so maybe `$` is common/standardized and we should stick with just that?